docs/integrations/sources/shopify-migrations.md

```diff
@@ -1,5 +1,43 @@
 # Shopify Migration Guide
 
+## Upgrading to 1.2.0
```

Comment:
let's bump to 2.0.0 since this is a breaking change

Reply:
Done.